src/index.html

```diff
@@ -4,10 +4,302 @@
+              <span class="testimonials__position">Senior Vice President</span>
+              <p class="paragraph testimonials__paragraph">
+                My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart
+              </p>
+            </div>
+            <div class="testimonials__person">
+              <img
+                src="images/people/3.jpg"
+                alt="a middle-aged man with thick beard wearing shirt and glasses"
+                class="testimonials__photo"
+              >
+              <h3 class="paragraph-heading testimonials__name">Max Henry</h3>
+              <span class="testimonials__position">Chief Design Officer</span>
+              <p class="paragraph testimonials__paragraph">
+                When my lovely valley of steam rises and half-day sun is above an impermeable
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="main__section">
+        <div class="contact-us container" id="contact-us">
+          <h2 class="section-heading contact-us__heading">Contact us</h2>
+          <div class="contact-us__content">
+            <div class="contact-us__item">
+              <div class="contact-us__contacts">
+                <div class="contact-us__phone">
+                  <span class="contact-us__title">Phone</span>
+                  <a href="tel:+1 234 5555-55-55" class="contact-us__link">+1 234 5555-55-55</a>
+                </div>
+                <div class="contact-us__email">
+                  <span class="contact-us__title">Email</span>
+                  <a href="mailto:hello@miami.com" class="contact-us__link">hello@miami.com</a>
+                </div>
+              </div>
+              <div class="contact-us__address">
+                <span class="contact-us__title">Address</span>
+                <p class="paragraph contact-us__paragraph">400 first ave. n. <br>suite 700 Minneapolis, MN 55401</p>
+              </div>
+            </div>
+            <form
+              action="http://www.example.com"
+              class="form"
+              method="post"
+            >
+              <input
+                type="text"
+                name="contact-us__name"
+                id="contact-us__name"
+                class="form__item"
+                placeholder="Name"
+                required
+              >
+              <input
+                type="email"
+                name="contact-us__email"
+                id="contact-us__email"
+                class="form__item"
+                placeholder="Email"
+                required
+              >
+              <textarea
+                class="form__item form__item--textarea"
+                name="contact-us__textarea"
+                id="contact-us__textarea"
+                placeholder="Message"
+              >
+              </textarea>
```

Comment:
```suggestion
              <textarea
                class="form__item form__item--textarea"
                name="contact-us__textarea"
                id="contact-us__textarea"
                placeholder="Message"
             ></textarea>
```
ось так має з'явитись placeholder